driver/makedepend.ml

```diff
@@ -578,7 +578,7 @@ let main () =
      "-all", Arg.Set all_dependencies,
         " Generate dependencies on all files";
      "-allow-approx", Arg.Set allow_approximation,
-        " Fallback to a lexer-based approximation on unparseable files";
+        " Fallback to a lexer-based approximation on unparsable files";
```

Comment:
I think both of these are correct spellings?

Reply:
So let's take the codespell automatic correction...

Reply:
According the venerable wiktionary (I’m not on the network with access to the OED), dropping the ‘e’ is preferred in formal writing (cf. movable, etc.). Apparently I’m not a formal person...